src/main/java/org/fenixedu/academic/bootstrap/FenixBootstrapper.java

```diff
@@ -154,6 +156,29 @@
         return Lists.newArrayList();
     }
 
+    private static void createEvaluationSeasons() {
+
+        LocalizedString acronym = new LocalizedString.Builder().with(Locale.forLanguageTag("pt-PT"), "AE")
+                .with(Locale.forLanguageTag("en-GB"), "SA").build();
+        new EvaluationSeason(acronym, BundleUtil.getLocalizedString(Bundle.ENUMERATION, "SPECIAL_AUTHORIZATION"), false, false,
```

Comment:
With the previous auxillay method, these lines would be replaced by something like: 
`createEvaluationSeasons("AE", "SA", "SPECIAL_AUTHORIZATION", false, false, true, false);`